### PR TITLE
Allow user to view event within selected time range

### LINKED ISSUE #588: [FEATURE] Filter to show events within a time range
on event page, let users to enter the startdate and enddate to specify the time range search. All satisfied events will be shown on the page.

@clav1d you can start work on the backend function to query events based on the user input, e.g., startdate, enddate.

### PR DESCRIPTION
**Pull request by clav1d**
close #588

### CODE REVIEW

File: templates/events/select_status.html [hunk 1]
```diff
@@ -40,4 +40,21 @@
             location.reload(true);
         });
     })
+
+    $("#from, #to").on('change', function() {
+            let from =  document.getElementById('from').value;
```
Comment (bingzhang) [inline templates/events/select_status.html:45]: do the time conversion here.

**Comment by bingzhang** [comment]
@clav1d did you add reset datetime button? I run this pr but cannot find that button on GUI.

**Comment by clav1d** [comment]
> @clav1d did you add reset datetime button? I run this pr but cannot find that button on GUI.

Not yet, I'll make it around tomorrow.

**Comment by bingzhang** [comment]
@clav1d can you fix the conflicts, please?

**Review by bingzhang — APPROVED** [review]

**Pull request merged**